packages/protocol/test/common/metatransactionwallet.ts

```diff
@@ -354,6 +370,29 @@ contract('MetaTransactionWallet', (accounts: string[]) => {
             assertEqualBN(await wallet.nonce(), 1)
           })
         })
+
+        describe('when lengths in length array do not match up with lengths in data', async () => {
+          it('reverts with correct message', async () => {
```

Comment:
where are we checking the revert message?

Reply:
I was planning on it, but noticed we don't check messages, then forgot to change my description back. Perhaps it's something we SHOULD do though..